templates/publisher/publicise/github_badges.html

```diff
@@ -63,19 +58,12 @@ <h4>Promote your snap using embeddable GitHub badge</h4>
         <label>Markdown:</label>
       </div>
       <div class="col-7">
-        <div class="p-code-copyable not-cli">
-          <code class="p-code-copyable__input" id="snippet-badge-markdown">[![{{ snap_title }}](https://snapcraft.io/{{ snap_name }}/badge.svg)](https://snapcraft.io/{{ snap_name }})</code>
-          <button class="p-code-copyable__action js-clipboard-copy" data-clipboard-target="#snippet-badge-markdown">Copy to clipboard</button>
-        </div>
+        <pre style="white-space: pre-line;"><code id="snippet-badge-markdown">[![{{ snap_title }}](https://snapcraft.io/{{ snap_name }}/badge.svg)](https://snapcraft.io/{{ snap_name }})</code></pre>
```

Comment:
When it comes to wrapping - code blocks will not wrap by default.

We are adding wrapping to code snippet ([in next version of Vanilla](https://github.com/canonical-web-and-design/vanilla-framework/pull/3502)) via `is-wrapped` class on `p-code-snippet__block`. So - this can be like that for now, but if you want to make it 'ready' for next version of Vanilla you need to replace `<pre><code>` with `<div class="p-code-snippet"><pre class="p-code-snippet__block is-wrapped`. And either wait for next version to introduce wrapping, or add custom style for `.p-code-snippet__block.is-wrapped { white-space: pre-wrap }`.